x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/actions_log_table.tsx

```diff
@@ -129,11 +130,13 @@ const getResponseActionListTableColumns = ({
           <StyledFacetButton
             icon={
               <EuiAvatar
+                title={undefined}
```

Comment:
```suggestion
                // This information is already displayed by the subsequent text & EuiToolTip
                // no need to repeat it for SR or mouse users
                aria-hidden={true}
                title=""
```

Reply:
Thanks @cee-chen. I've made changes as you suggested. Please have a look.

Reply:
Changes look great. Nice job Ash!